ChiModules/LinearBoltzmannSolver/lbs_03e_tgdsa.cc

```diff
@@ -167,13 +167,13 @@ void LinearBoltzmann::Solver::DisAssembleTGDSADeltaPhiVector(LBSGroupset& groups
     auto& transport_view = cell_transport_views[c];
 
     int xs_id = matid_to_xs_map[cell.material_id];
-    std::vector<double>& xi_g = material_xs[xs_id]->xi_Jfull_g;
+    std::vector<double>& xi_g = material_xs[xs_id]->xi_Jfull;
```

Comment:
this name does not speak to me. what is xi_Jfull ? Please have this more readable. I am fine with more renaming.

Reply:
This is going to move out of the transport xs class with issue #148. Let us collaborate on this then.